consensus/propagation/reactor.go

```diff
@@ -96,8 +94,8 @@ func NewReactor(
 		privval:       config.Privval,
 		chainID:       config.ChainID,
 		BlockMaxBytes: config.BlockMaxBytes,
-		partChan:      config.PartChan,
-		proposalChan:  config.ProposalChan,
+		partChan:      make(chan types.PartInfo, 2500),
+		proposalChan:  make(chan types.Proposal, 100),
```

Comment:
Why such values? Shouldn't this be a configurable, or a part depend on block size (`partChan`) and number of validators (`proposalChan`)?

Reply:
we felt that 2500 is good enough to hold more than a full 128mb (2048 parts) block parts. And we don't want more because having this many parts not processed by the consensus reactor only shows there is an issue somewhere.

For 100, that's 100 new proposal, we shouldn't need that much anyway.

We can make them configurable, but unless we want to change them frequently, I don't think it's worth doing that. Also, they're not breaking or anything, so we can make them configurable anytime.